src/platyPS/platyPS.psm1

```diff
@@ -191,7 +191,14 @@ function New-MarkdownHelp
                     $online = $OnlineVersionUrl
                 }
 
-                $commandName = $mamlObject.Name
+                if ($mamlObject.Name -like "*.ps1")
+                {
+                    $commandName = ".\$($mamlObject.Name)"
```

Comment:
Not following, what the resulting $commandName would be for ps1 file?

Reply:
Get-Command requires a path to realize it's talking about a script; adding a .\ to the beginning of $commandName bypasses the original error I opened the issue for.

Reply:
...and this just made me realize that this only works when you're running New-MarkdownHelp from the same directory as the ps1. I'll rethink.